automatic/musescore/update.ps1

```diff
@@ -1,6 +1,6 @@
 ﻿Import-Module AU
 
-$releases = 'https://musescore.org/en/download/musescore.msi'
+$releases = 'https://musescore.org/en/download'
```

Comment:
This change is invalid, without additional work done in the `au_GetLatest` handler.

You are effectively breaking the current automatic updater by changing this URL without making any other changes.

Can you please make the necessary changes to make the updater work as well?

Reply:
Sorry for the confusion! I mixed things up here. For sure i did not intend to break the update-script. I corrected it now, on basis of the [disscussion with @pauby](https://github.com/orgs/chocolatey-community/discussions/2325). Would be great to have it merged. Sorry for not replying sooner - i was on vacation ;)